### PR TITLE
symbolized land ownership for Q

### PR DESCRIPTION
**Pull request by lauren-herbine**
Land ownership should be symbolized the same as Arc (aside from transparency) now. I also added USACE symbology, which isn't currently in Arc

### CODE REVIEW

**Comment by lauren-herbine** [comment]
Should look like this:
image

**Review by shelbysawyer — APPROVED** [review]

File: Symbology/qgis/RSContext/ownership.qml [hunk 1]
```diff
@@ -0,0 +1,753 @@
...
+    <categories>
+      <category label="Fish and Wildlife Service" value="FWS" symbol="0" render="true"/>
+      <category label="Other" value="UND" symbol="1" render="true"/>
+      <category label="US Army Corps of Engineers" value="USACE" symbol="2" render="true"/>
+      <category label="US Forest Service" value="USFS" symbol="3" render="true"/>
+      <category label="Private" value="PVT" symbol="4" render="true"/>
+      <category label="National Park Service" value="NPS" symbol="5" render="true"/>
+      <category label="Bureau of Land Management" value="BLM" symbol="6" render="true"/>
+      <category label="Bureau of Indian Affairs" value="BIA" symbol="7" render="true"/>
+      <category label="State" value="ST" symbol="8" render="true"/>
```
Comment (joewheaton) [inline Symbology/qgis/RSContext/ownership.qml:25]: @lauren-herbine can you control the display order in the legend of these? If so, is it simply by changing this `symbol` key? Perhaps we should make the order in the legend a little clearer?

**Review by joewheaton — CHANGES_REQUESTED** [review]
Please see my minor changes suggested on display order to match what we do in Arc?

image

**Comment by lauren-herbine** [comment]
@joewheaton I've played around with the .qml file a bit, and so far neither changing the symbol nor changing the actual order seems to have an affect on the order in the legend:

```
  <renderer-v2 enableorderby="0" type="categorizedSymbol" symbollevels="0" forceraster="0" attr="ADMIN_AGEN">
    <categories>
      <category label="Private" value="PVT" symbol="1" render="true"/>
      <category label="National Park Service" value="NPS" symbol="2" render="true"/>
      <category label="Fish and Wildlife Service" value="FWS" symbol="3" render="true"/>
      <category label="US Forest Service" value="USFS" symbol="4" render="true"/>
      <category label="Other" value="UND" symbol="5" render="true"/>
      <category label="State" value="ST" symbol="6" render="true"/>
      <category label="Bureau of Land Management" value="BLM" symbol="7" render="true"/>
      <category label="Bureau of Indian Affairs" value="BIA" symbol="8" render="true"/>
      <category label="US Army Corps of Engineers" value="USACE" symbol="9" render="true"/>
    </categories>
```

I'll do some research. @shelbysawyer tagging for future reference

**Review by joewheaton — APPROVED** [review]

**Pull request merged**